src/input/__tests__/__snapshots__/input.test.js.snap

```diff
@@ -3,6 +3,8 @@
 exports[`Input - basic functionality: input has correct props 1`] = `
 Object {
   "aria-invalid": false,
+  "aria-label": undefined,
```

Comment:
do we want to provide a value for the tests?

Reply:
updated, should have default `null`

Reply:
+ updated e2e setting